src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/DeleteResourcesValidation.feature.disabled

```diff
@@ -0,0 +1,37 @@
+# This is a rough draft feature for future use.
+@ignore
+Feature: Resources "Delete" Operation validations
+
+        Background:
+            Given the Data Management Service must receive a token issued by "http://localhost"
+              And user is already authorized
+              And request made to "/ed-fi/absenceEventCategoryDescriptors" with
```

Comment:
For consistency, suggestion: ` a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with`

Reply:
Sure, also I think we need to consider the Feature files that already exist and make sure we are following a standard in all of them

Reply:
Yes, completely agree.

Reply:
I have created a ticket to address this
[DMS-183](https://tracker.ed-fi.org/browse/DMS-183)